src/EPub.ts

```diff
@@ -0,0 +1,244 @@
+import Store from "./Store";
+
+type xmlObject = {
+  [key: string]: string[] | string | xmlObject | [];
+};
+
+function xmlToJson(xml: any) {
```

Comment:
does DomParser not work here? (https://developer.mozilla.org/en-US/docs/Web/API/DOMParser)

Reply:
This function is transforming the output from DomParser into JSON. It looks like the MIME types that are supported by DomParser are: 

text/html | Document
-- | --
text/xml | XMLDocument
application/xml | XMLDocument
application/xhtml+xml | XMLDocument
image/svg+xml | XMLDocument

 https://github.com/NYPL-Simplified/webpub-viewer/blob/6a4325c2ee3c836e0a9af413ef1f25640121e8e3/src/Manifest.ts#L86